src/transform/src/lib.rs

```diff
@@ -206,6 +206,7 @@ impl Default for FuseAndCollapse {
                 // `MirRelationExpr::Constant` if that is the case, so that subsequent use can
                 // clearly see this.
                 Box::new(crate::reduction::FoldConstants { limit: Some(10000) }),
+                Box::new(crate::fusion::top_k::TopK),
```

Comment:
Can this be moved earlier in the loop? `FoldConstants` is always supposed to be last.

Reply:
right, done